app/(components)/TicketCard.jsx

```diff
@@ -20,12 +20,12 @@ const TicketCard = ({ ticket }) => {
   }
 
   const createdDateTime = formatTimestamp(ticket.createdAt);
+  const updatedDateTime = formatTimestamp(ticket.updatedAt);
 
   return (
     <div
-      className={`flex flex-col hover:bg-card-hover bg-card rounded-md shadow-lg p-3 m-2 ${
-        ticket.priority > 4 ? 'bg-red-500' : ''
-      }`} 
+      className={`flex flex-col hover:bg-card-hover bg-card rounded-md shadow-lg p-3 m-2 ${ticket.priority > 4 ? 'bg-red-500' : ''
```

Comment:
please refrain from committing unnecessary refactoring changes,
we would like this to be regressed